dask-sql/dask-sql.ipynb

```diff
@@ -44,6 +45,7 @@
     "from dask.distributed import Client\n",
     "\n",
     "client = Client(cluster)\n",
+    "client.wait_for_workers()\n",
```

Comment:
`n_workers=0` by default for `Client.wait_for_workers`. Do we want to to change this to `client.wait_for_workers(10)`?

Reply:
Ooops, good catch, thanks @jrbourbeau . I think the bigger issue here was the too-low-memory in `dask-sql`, but I should also make this update.